app/routes/events/components/EventEditor/EditorSection/Header.tsx

```diff
@@ -65,6 +66,14 @@ const Header = ({
         className={styles.formField}
       />
 
+      {values.pinned && (
+        <Card style={{ marginTop: 0 }} severity="warning">
+          <span>
+            OBS! Du må ha godkjenning fra ledelsen for å feste til forsiden.
+          </span>
```

Comment:
See other comments

Reply:
Juni and I concluded that the syling wasn't necessary so we removed it:)